test/e2e/testing-manifests/ingress/nginx/rc.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 0
       containers:
-      - image: gcr.io/google_containers/nginx-ingress-controller:0.8.3
+      - image: gcr.io/google_containers/nginx-ingress-controller:0.9.0-beta.1
```

Comment:
where is this defined? https://github.com/kubernetes/contrib/blob/master/ingress/controllers/nginx/Makefile#L4 is still 0.8.3.